package.json

```diff
@@ -51,6 +52,7 @@
     "babel-loader": "^6.2.0",
     "babel-plugin-add-module-exports": "^0.1.2",
     "babel-plugin-transform-dev-warning": "^0.1.0",
+    "babel-plugin-transform-object-assign": "git://github.com/newoga/babel-plugin-transform-object-assign.git",
```

Comment:
Do you have a preference as to what we call it?
`babel-plugin-transform-inject-object-assign`
`babel-plugin-transform-replace-object-assign`
`{any other ideas}`